README.rst

```diff
@@ -68,6 +70,71 @@ Requirements
 If using postgres as database driver, ensure the `postgres contrib`_ package is
 installed on server side.
 
+Recomended usage
+================
+
+Given target minion_id is 'srv'.
+
+/srv/salt/top.sls::
+
+    base:
+        'srv'
+            - saemref
+            - saemref.supervisor
+
+
+/srv/pillar/top.sls::
+
+    base:
+        'srv':
+            - saemref
+
+
+/srv/pillar/saemref.sls::
+
+    saemref:
+        lookup:
+            instance:
+                base_url: http://saemref.example.com
+                # generated with: pwgen -sy 32
+                sessions_secret: "iajI*=^m;oqNc7:%ta'A6r3BDLUjB^,y"
+                authtk_session_secret: ")E6"LT,XMUd."Ekg+(<2m%RVqLxdV[a4"
+                authtk_persistent_secret: "Ay]%v4w>B+yP$|~BLTL^+785(6@=,Gk$"
+                anonymous_user: anon
+                anonymous_password: anon
+            db:
+                driver: postgres
+                host: pg.example.com
+                port: 5432
+                name: saemref
+                user: saemref
+                pass: uOCWIzj4
+            admin:
+                login: admin
+                pass: qj0FOtM6UC
+
+    postgres:
+        version: 9.4
+
+
+Example of reverse proxy configuration for nginx::
+
+    server {
+        listen 80;
+        server_name saemref.example.com;
+        location / {
+            proxy_pass http://srv:8080;
+        }
+        location /oai {
+            proxy_pass http://srv:8081;
+        }
+    }
+
+
+For the first installation run ``salt srv state.sls db-create`` to create
+the database (WARNING: this will destroy existing database if exists).
+
+Then run: ``salt srv state.highstate`` to finish the installation.
```

Comment:
`highstate` never run `db-create`, so running highstate before would try to start the instance without database.